sr_hand_health_report/src/sr_hand_health_report/backlash_check.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python3
+
+# Copyright 2022-2023 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import time
+import math
+import rospy
+from sr_hand_health_report.sr_hand_health_report_check import SrHealthReportCheck
+import numpy as np
+from urdf_parser_py.urdf import URDF
+
+
+class BacklashCheck(SrHealthReportCheck):
+
+    TEST_COUNT = 10
+    THRESHOLD = 0.01
+    FINGER_PWM = -250
+    WIGGLING_TIME = 2
+    FINGERS = ("TH", "FF", "MF", "RF", "LF")
+    PASSED_THRESHOLDS = {'std': 0.015, 'avg': 0.01}
+
+    def __init__(self, hand_side, fingers_to_test):
+        """
+            Initialize the BacklashCheck object
+            @param hand_side: String indicating the side
+            @param fingers_to_test: List of finger prefixes to test
+        """
+        super().__init__(hand_side, fingers_to_test)
+        self._side_sign_map = {"ff": -1, "mf": -1, "rf": 1, "lf": 1, "th": 1, "wr": 1}
```

Comment:
isn't this already defined in `SrHealthReportCheck`?

Reply:
If you're changing this (to define it in one place), consider renaming it to include `j4` to make it clearer what it's for on first glance.

Reply:
Renamed to _j4_side_sign_map and removed from Backlash class [d24cf99](https://github.com/shadow-robot/sr_tools/pull/66/commits/d24cf99dd59872e7721f20402fb26b636719784a)